### DISCUSSION TITLE
Empresas usando o Projeto

### DISCUSSION THREAD

**Discussion by augustresende** [BrasilAPI/BrasilAPI#220] [Q&A]
Seria legal ter no README uma lista das empresas que usam o projeto. Grandes projetos costumam ter isso para mostrar o uso real que os projetos tem. O que acham?

**Comment by iamchicao** [comment]
@AugustoResende Também acho super válido e de grande importância, posso ajudar também!

**Comment by lucianopf** [comment]
A ideia é excelente mano!! 😬 

Tem algumas frentes que acho que a gnt poderia tocar vejam se faz sentido pfv 🙏 😬 

1 - Usar essa issue para que as empresas ativamente comentem e peçam pra serem incluídas numa lista de empresas usando no README.

2 - De tempos em tempos consultar os logs e trazer um agregado das urls de origem da requisição e inferir as empresas usando. 



Sobre esse segundo ponto tenho umas novidades que comecei a explorar ontem usando o Datadog:

 
<img width="259" alt="Screen Shot 2020-11-18 at 17 53 42" src="https://user-images.githubusercontent.com/8251208/99586852-0ab81900-29c7-11eb-96f3-6f4703dd315b.png">

**Comment by RodriAndreotti** [comment]
Rapaz, tem bastante gente usando heim, se esse c6 for o c6bank é uma empresa razoavelmente grande.
